pkg/test/resourcefixture/testdata/basic/bigtable/v1beta1/bigtableinstance/bigtableinstanceautoscaling/update.yaml

```diff
@@ -24,17 +24,17 @@ spec:
     zone: us-central1-a
     autoscalingConfig:
       cpuTarget: 70
-      maxNodes: 20
+      maxNodes: 5
       minNodes: 2
   - clusterId: cluster-2-${uniqueId}
-    zone: us-west1-b
+    zone: us-central1-b
     autoscalingConfig:
       cpuTarget: 70
-      maxNodes: 20
+      maxNodes: 5
       minNodes: 2
   - clusterId: cluster-3-${uniqueId}
-    zone: us-east1-d
+    zone: us-central1-c
```

Comment:
We might need a README on this test.  We were testing was whether we can insert a cluster into "the middle" of the cluster list (vs appending at the end).

Reply:
(Everything else looks fine, just reorder the last two clusters in the update.yaml here I think)

Reply:
Ah, ok, fixed!